test/setup/main.tf

```diff
@@ -26,9 +26,37 @@ module "project" {
   skip_gcloud_download = true
 
   activate_apis = [
+    "cloudkms.googleapis.com",
     "cloudresourcemanager.googleapis.com",
     "bigquery.googleapis.com",
     "bigquerystorage.googleapis.com",
     "serviceusage.googleapis.com"
   ]
 }
+
+#Can't delete this with Terraform.
+module "kms_keyring" {
+  source     = "terraform-google-modules/kms/google"
+  version    = "~> 1.2"
+  depends_on = [module.project] #waits for API config
+
+  project_id      = module.project.project_id
+  location        = "us"
+  keyring         = "ci-bigquery-keyring"
+  keys            = ["foo"]
+  prevent_destroy = "false"
+}
+
+module "initialize_encryption_account" {
+  #pinned to prevent bug with random cache count evaluation, see #82
+  source     = "terraform-google-modules/gcloud/google"
+  version    = "1.2.0"
+  depends_on = [module.project] #waits for API config
```

Comment:
terraform-google-modules/terraform-google-gcloud#82 only seems to happen with `depends_on` for modules. We don't actuallly need this as we have implicit dependency via `create_cmd_body       = format("show --encryption_service_account --project_id %s", module.project.project_id)` and  ` module.project.project_id` is [only returned after the API is enabled](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/5886a4e4430f551266710d6f635860963be6a4ec/modules/core_project_factory/outputs.tf#L31).

Reply:
Removed.